### PR TITLE
Newsletter 13: Add gfx, wgpu, and vange-rs news

### PR DESCRIPTION
**Pull request by kvark**
_Part of #232 ("Newsletter 13")_

### CODE REVIEW

**Review by 17cupsofcoffee — COMMENTED** [review]
Content looks good, just needs line breaks adding on lines 70-73 to satisfy the linter. I can't figure out how to do multi-line suggestions on GitHub, but you can just copy and paste this if you want:

```md
See [video on /r/rust_gamedev][vangers-shadow-video] and technical description
on the [Hybrid Shadows][vangers-shadow-blog] post of the blog.

Another exciting development - the new bruteforce rendering technique allowing
to shift the camera behind the mechos as in 3rd person view.

See [video on /r/rust_gamedev][vangers-bars-video] and technical description on
the [Bar Painting][vangers-bars-blog] post of the blog.
```

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -62,10 +62,43 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
 ## Library & Tooling Updates
 
+### [gfx-rs][gfx-repo] and [wgpu][wgpu-site]
+
+![wgpu-rs water](https://wgpu.rs/screenshots/example-water.gif)
+
+gfx-rs project and wgpu ecosystem have observed the release of 0.6 versions :tada:
+Read about the details on [gfx-rs blog](https://gfx-rs.github.io/2020/08/18/release-0.6.html).
+
+[@sothr](https://github.com/sothr) has reworked the [wgpu instancing tutorial][wgpu-instancing-tutor]. See discussion at [/r/rust_gamedev][wgpu-instancing-reddit].
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:95]:
```suggestion
[@sothr](https://github.com/sothr) has reworked the
[wgpu instancing tutorial][wgpu-instancing-tutor]. See discussion at
[/r/rust_gamedev][wgpu-instancing-reddit].
```

Whoops, the linter was also complaining about this one 😅

**Review by 17cupsofcoffee — APPROVED** [review]
🎉

**Review by ozkriff — APPROVED** [review]
Embedded images, made a few style/fmt tweaks, and added a short overview of gfx/wgpu updates.

**Comment by ozkriff** [comment]
@kvark are you fine with the edits?

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by kvark** [comment]
shipit!

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**